api/src/main/java/jakarta/enterprise/inject/spi/ProcessManagedBean.java

```diff
@@ -38,4 +41,16 @@ public interface ProcessManagedBean<X> extends ProcessBean<X> {
      * @throws IllegalStateException if called outside of the observer method invocation
      */
     public AnnotatedType<X> getAnnotatedBeanClass();
+
+    /**
+     * Returns a new {@link InvokerBuilder} for given method. The builder eventually produces an invoker
+     * for the given method.
+     * <p>
+     * The {@code method} must belongs to the bean being registered, otherwise an exception is thrown.
```

Comment:
```suggestion
     * The {@code method} must belong to the bean being registered, otherwise an exception is thrown.
```

Reply:
I'll reword this based on feedback above, so there will be no "belong" anymore.